jdaviz/__init__.py

```diff
@@ -6,6 +6,9 @@
 from ._astropy_init import *   # noqa
 # ----------------------------------------------------------------------------
 
+from glue_jupyter.bqplot.profile import layer_artist
+layer_artist.USE_GL = False
+
```

Comment:
DO NOT MERGE this either (but let's us test with just a bqplot implementation without having to worry about LinesGL or MPL)

Reply:
This can be removed if glue is changed to either expose the class (in which case we can avoid a lot of the hackiness in init) or to `import bqplot_image_gl` and set `LinesClass = bqplot_image_gl.LinesGL` instead of the current `from bqplot_image_gl import LinesGL`.